spec/features/items/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor, " do
+  before :each do
+    @user_1 = User.create!(name: 'User 1', email: 'user_1@email.com', password_digest: 'password1', active: true, address: '1 Street', city: 'City 1', state: 'CO', zip: '00001', role: 1)
```

Comment:
We should be giving the user a password, not a password_digest. Same goes for both of these.

Reply:
We should give them role of "merchant" instead of role 1.